test/test_cuda.py

```diff
@@ -24,8 +24,13 @@
 
 
 def is_floating(t):
-    return type(t) in [torch.FloatTensor, torch.DoubleTensor,
```

Comment:
This function will silently always return False if you pass it a tensor rather than a tensor type. Maybe it should check `isinstance(t, type)` and raise an error if this is not the case?